.env.example

```
@@ -38,9 +38,9 @@ MIRU_GH_TOKEN='github-oauth-client-secret'
 STRIPE_PUBLISHABLE_KEY="stripe_publishable_key"
 STRIPE_SECRET_KEY="stripe_secret_key"
 STRIPE_WEBHOOK_ENDPOINT_SECRET="stripe_webhook_endpoint_secret"
-STRIPE_SUBSCRIPTION_PRICE_ID="price_fallback"
-STRIPE_SUBSCRIPTION_PRICE_ID_MONTHLY="price_monthly"
-STRIPE_SUBSCRIPTION_PRICE_ID_YEARLY="price_yearly"
+STRIPE_SUBSCRIPTION_PRICE_ID=""
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)